lib/_tls_wrap.js

```diff
@@ -793,6 +795,15 @@ TLSSocket.prototype.getCertificate = function() {
   return null;
 };
 
+TLSSocket.prototype.enableTrace = function(unused) {
+  if (arguments.length !== 0)
+    throw new ERR_INVALID_ARG_VALUE('unused', unused);
```

Comment:
Why is all of this added if there are no parameters?

Reply:
Maybe overkill, tell me if it is, I'll delete it. My thinking is that even though I could never have debugged the TLS1.3 development without this feature as it is, so I believe its really useful, I'm not really happy that the trace is written directly to stderr. A future iteration might take an argument that makes the trace output go somewhere else, but to do that and not be semver-major, we might need to be _really_ sure that no current user is passing stray arguments to this API.

Reply:
The same could be said of any other public function that has added parameters over time? I think we should implement the functionality or remove the unused parameter and checking logic, otherwise it's confusing.